### PR TITLE
DOC-900: Creds overhaul for S

### PR DESCRIPTION
**Pull request by freakwriter**
Updated all S* credentials to template, except for:

- S3: Needs a larger overhaul of both the cred and the app node; I have this logged as more complex, bumped to separate issue: https://linear.app/n8n/issue/DOC-842/review-more-complex-creds-pages
- Segment: I believe their API endpoint has been updated and both the terminology and expectations we have here are wrong. Added some more notes about this into the Google Sheet where I'm tracking issues.

Note that this includes renaming sms77 to seven to match what's already in the app, but as the in-app docs link still points to the sms77 URL, I'm not sure if that was correct. I isolated the renaming and URL changes to [their own commit](https://github.com/n8n-io/n8n-docs/commit/69950ea355f15d491c05c6f1b16b5517f83cea26) in case we need to rollback.

This branch also includes one new snippet to standardize the introductory credential statement we use for credential-only nodes; I'll need to retroactively add that to already-released changes but wanted to get it out there now.

### CODE REVIEW

**Comment by freakwriter** [comment]
@StarfallProjects I've updated to make sure we keep the sms77 file names, but otherwise everything is set up for seven. I think that's the big change I needed to make here before full review, but I'm sure you'll find some other hidden delights...

File: docs/integrations/builtin/credentials/securityscorecard.md [hunk 1]
```diff
@@ -6,29 +6,28 @@ contentType: integration
...
-<iframe width="840" height="472.5" src="https://www.youtube.com/embed/0g1YLkNBfnM" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-</div>
+Refer to [SecurityScorecard's Developer documentation](https://securityscorecard.readme.io/docs/integrate-ratings-platform-services){:target=_blank .external-link} and [API documentation](https://securityscorecard.readme.io/reference/introduction){:target=_blank .external-link} for more information about the service.
+
+## Using API key
+
+To configure this credential, you'll need:
+
+- An **API Key**: Generate an API key in one of two ways:
+    1. As a user in [**My Settings > API**](https://platform.securityscorecard.io/#/my-settings/api){:target=_blank .external-link}. Refer to [Get an API key](https://securityscorecard.readme.io/docs/getting-started#step-1-get-an-api-key){:target=_blank .external-link} for more information.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/securityscorecard.md:31]:
```suggestion
    * As a user in [**My Settings > API**](https://platform.securityscorecard.io/#/my-settings/api){:target=_blank .external-link}. Refer to [Get an API key](https://securityscorecard.readme.io/docs/getting-started#step-1-get-an-api-key){:target=_blank .external-link} for more information.
```

File: docs/integrations/builtin/credentials/securityscorecard.md [hunk 2]
```diff
@@ -6,29 +6,28 @@ contentType: integration
...
+    2. As a bot user: View the bot user and select **create token**. Refer to [Authenticate with a bot user](https://securityscorecard.readme.io/docs/authentication#){:target=_blank .external-link} for more information.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/securityscorecard.md:32]:
```suggestion
    * As a bot user: View the bot user and select **create token**. Refer to [Authenticate with a bot user](https://securityscorecard.readme.io/docs/authentication#){:target=_blank .external-link} for more information.
```

File: docs/integrations/builtin/credentials/shopify.md [hunk 3]
```diff
@@ -13,29 +13,50 @@ You can use these credentials to authenticate the following nodes with Shopify.
 
 ## Prerequisites
 
-Create a [Shopify](https://shopify.com/){:target=_blank .external-link} account.
-
-Then [create a custom app](https://help.shopify.com/en/manual/apps/app-types/custom-apps){:target=_blank .external-link} to authenticate to.
+- Create a [Shopify](https://shopify.com/){:target=_blank .external-link} account.
+- [Create and install a custom app][(https://help.shopify.com/en/manual/apps/app-types/custom-apps){:target=_blank .external-link}].
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/shopify.md:17]:
```suggestion
- [Create and install a custom app](https://help.shopify.com/en/manual/apps/app-types/custom-apps){:target=_blank .external-link}.
```

**Review by StarfallProjects — APPROVED** [review]

**Pull request merged**